examples/example3/README.md

```diff
@@ -0,0 +1,32 @@
+
+# Example 3
+
+This is a simple example to show how DiffSync can be used to compare and synchronize data with a remote system like via a REST API like Nautobot.
+
+For this example, we have a shared model for Region and Country defined in `models.py`.
+A Country must be associated with a Region and can be part of a Subregion too.
```

Comment:
I'm not sure the Subregion adds much to this example, and I find it a bit confusing - it looks like a "Subregion" is just used as the `description` of the constructed Country `region`? But Nautobot supports nesting of `region`s, so why not assign the Country directly to the most relevant "sub-"`region` instead of the more general base `Region`?

Reply:
Somewhat agreed, I was just looking for a field to update that is not part of the identifier ...  
I'll update the example to create a custom field under dcim.site to store the population per country

Reply:
Sounds good, I think that'll be clearer. Thanks!